tests/integration/test_sca/test_basic.py

```diff
@@ -0,0 +1,205 @@
+'''
+copyright: Copyright (C) 2015-2023, Wazuh Inc.
+           Created by Wazuh, Inc. <info@wazuh.com>.
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: These tests will check if the `enabled` option of the SCA module
+       is working correctly. This option is located in its corresponding section of
+       the `ossec.conf` file and allows enabling or disabling this module.
+
+components:
+    - sca
+
+targets:
+    - manager
+    - agent
+daemons:
+    - wazuh-modulesd
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Ubuntu Focal
+    - Ubuntu Bionic
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/capabilities/sec-config-assessment/index.html
+
+tags:
+    - sca
+'''
+import os
+import pytest
+
+from wazuh_testing.constants.paths.logs import WAZUH_LOG_PATH
+from wazuh_testing.utils import callbacks, configuration
+from wazuh_testing.tools.monitors import file_monitor
+from wazuh_testing.modules.sca import patterns
+from wazuh_testing.modules.modulesd.configuration import MODULESD_DEBUG
+
+
+pytestmark = [pytest.mark.linux, pytest.mark.tier(level=0)]
+
+# Reference paths
+TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
```

Comment:
Use `Path` instead of `os.path`:
```
from pathlib import Path
```

Reply:
Besides, `data`, `configuration_templates` and `test_cases` folders should be defined in the `__init__.py` file. Check other tests as an example.

Reply:
Repeat in all tests.